user-console/src/main/java/org/pentaho/mantle/client/commands/ImportFileCommand.java

```diff
@@ -87,6 +89,10 @@ public void okPressed() {
     };
     importDialog.setCallback( callback );
     importDialog.center();
+    Focusable focusable = ElementUtils.findFirstKeyboardFocusableDescendant( importDialog.getForm().getWidget() );
+    if( focusable != null ){
+      focusable.setFocus( true );
+    }
```

Comment:
The default dialog code should already place the focus on the "browse" button.
A bug that cause this to fail when the first element was disabled (your case) has been fixed already.
Please, rebase on `wcag` latest and retest.

Reply:
Upon pulling in latest code and testing, found that without this piece of code, focus is placed on the "cancel" button replicating the behaviour of the bug.

Reply:
Please see https://github.com/pentaho/pentaho-commons-gwt-modules/pull/843 which addresses this bug.

Reply:
Please see https://github.com/pentaho/pentaho-commons-gwt-modules/pull/843 which addresses this bug.